lib/module.js

```diff
@@ -101,6 +101,7 @@ function registerSSRHook (options) {
     const usedTags = detectTags(tags, params.APP)
 
     params.HEAD += usedTags
+      .filter((t, index, self) => index === self.findIndex((t2) => (t.url === t2.url))) //filter duplicates
```

Comment:
I think it would be better to move this filter into `detectTags` function because this is the responsibility of `detectTags`